confmap/confmaptest/configtest_test.go

```diff
@@ -30,11 +30,16 @@ func TestLoadConf(t *testing.T) {
 	assert.Equal(t, map[string]any{"floating": 3.14}, cfg.ToStringMap())
 }
 
+func TestToStringMapSanitizeNil(t *testing.T) {
+	cfg, err := LoadConf(filepath.Join("testdata", "nil.yaml"))
+	require.NoError(t, err)
+	assert.Equal(t, map[string]any{"slice": nil}, cfg.ToStringMap())
+}
+
 func TestToStringMapSanitizeEmptySlice(t *testing.T) {
```

Comment:
```suggestion
func TestToStringMapEmptySlice(t *testing.T) {
```

The test name should change